apps/src/lib/kits/maker/ui/SetupStep.jsx

```diff
@@ -0,0 +1,97 @@
+import React, {Component, PropTypes} from 'react';
+import color from '../../../../util/color';
+import FontAwesome from '../../../../templates/FontAwesome';
+
+export const HIDDEN = 'HIDDEN';
+export const WAITING = 'WAITING';
+export const ATTEMPTING = 'ATTEMPTING';
+export const SUCCEEDED = 'SUCCEEDED';
+export const FAILED = 'FAILED';
+export const CELEBRATING = 'CELEBRATING';
+export const STEP_STATUSES = [HIDDEN, WAITING, ATTEMPTING, SUCCEEDED, FAILED, CELEBRATING];
+
+const style = {
+  root: {
+    margin: '15px 0',
+  },
+  header: {
+    'fontSize': '26px',
+  },
+  body: {
+    margin: '15px 0 15px 40px',
+    fontSize: '14px',
+  }
+};
```

Comment:
these styles shouldn't be strings? I'm referring both to the keys and the values.